ott/core/sinkhorn_lr.py

```diff
@@ -392,12 +390,10 @@ def norm_error(self) -> Tuple[int]:
 
   def _converged(self, state: LRSinkhornState, iteration: int) -> bool:
     criterion, count_escape, i, tol = state.criterion, state.count_escape, iteration, self.threshold
```

Comment:
There's no attribute `.criterion`, only `.criterions`.

Reply:
Same goes for the `_diverged(self, state: LRSinkhornState, iteration: int) -> bool:` function.
`